.gitlab-ci-template.yml

```diff
@@ -34,15 +34,23 @@
   script:
     - cd synopsys-task
     - npm ci
-    - npm run all
+    - npm run format && npm run lint && npm run build && npm run package
 
-
-.code_coverage:
+.unit_test:
   image: gcr.io/snps-polaris-ops-prod/ember_cli:3.28.5-node_16.14
   script:
     - cd synopsys-task
     - npm ci
-    - npm test -- --coverage --collectCoverageFrom='src/**/*.{ts,jxs}'
+    - npm test > unit_test.log
+    - cat unit_test.log
+  artifacts:
+    paths:
+      - ./synopsys-task/unit_test.log
+
+.code_coverage:
+  image: gcr.io/snps-polaris-ops-prod/ember_cli:3.28.5-node_16.14
+  script:
+    - cat synopsys-task/unit_test.log
```

Comment:
can we combine these two jobs as doing same thing?
Ex: **unit_test_and_coverage**
we can anyways set coverage value with regex, we don't need to print twice.